cmd/eksctl-anywhere/cmd/checkimages.go

```diff
@@ -0,0 +1,92 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+
+	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/constants"
+	"github.com/aws/eks-anywhere/pkg/executables"
+	"github.com/aws/eks-anywhere/pkg/version"
+)
+
+type checkImagesOptions struct {
+	fileName string
+}
+
+var cio = &checkImagesOptions{}
+
+func init() {
+	rootCmd.AddCommand(checkImagesCommand)
+	checkImagesCommand.Flags().StringVarP(&cio.fileName, "filename", "f", "", "Filename that contains EKS-A cluster configuration")
```

Comment:
This would be a bit more flexible if instead of a cluster spec we took just the registry endpoint